### PR TITLE
feat: add/remove CIE repositories

### PR DESCRIPTION
**Pull request by astagi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,7 +356,11 @@ Pubblica Amministrazione con un'unica Identità Digitale da computer, tablet e s
 - [cieid-android-sdk](https://github.com/italia/cieid-android-sdk)
   <img align="right" src="https://img.shields.io/github/stars/italia/cieid-android-sdk?label=%E2%AD%90%EF%B8%8F&logo=github" alt="GitHub stars">
   <img align="right" src="https://img.shields.io/github/issues/italia/cieid-android-sdk" alt="GitHub issues">
-  
+  Android SDK for CIE
```
Comment (bfabio) [inline README.md:359]:
```suggestion
  iOS SDK for CIE
```

**Review by bfabio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (astagi) [reply README.md:359]: @bfabio this is the description for cieid-android-sdk, iOS SDK is below

Comment (bfabio) [reply README.md:359]: @astagi oops, sorry. I got misleaded by my own MD formatting 😅

**Review by bfabio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (astagi) [reply README.md:359]: no problem @bfabio :)

**Pull request merged**